src/LayoutComponents/nav/Nav.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import Breakpoint from '../Breakpoint.jsx';
+import Collapsible from './Collapsible.jsx';
+import SideNav from './SideNav.jsx';
+
+function Navigation() {
+
+  return (
+    <Breakpoint
+      render={
+        ({ TabletAndBelow, OnDesktop }) => {
+          return (
+            <div>
+              <OnDesktop>
+                <SideNav />
+              </OnDesktop>
+              <TabletAndBelow>
```

Comment:
This seems very unreliable when resizing. It works great using Chrome's device simulator, but not with Firefox's. It doesn't work when resizing my browser normally. I feel like it's important that this does work (example: want to put it side-by-side with my editor to manually copy something and play with it). It does seem to work consistently on page load.